src/input/mainArgs.ts

```diff
@@ -33,15 +33,76 @@ const commandParser = new ArgumentParser({
   add_help: false,
   parents: [dbArgs],
 });
+
 commandParser.add_argument("command", {
   help: "the command to run",
+  nargs: "?",
```

Comment:
command should still be required (this is just a special case where if you enter an unknown command or no command, then the help message displays.